global.R

```diff
@@ -37,19 +37,18 @@ for(i in dataset$Short_hand_code) {
 }
 
 # Add other factors to available plots
-dbFields <- dbListFields(database, "master_clinical")
-dbFields <- dbFields[-c(1,2,3,4)] # Removes UPN and other columns that can't be plotted
+dbFields <- dbListFields(database, "clinical")
+dbFields <- dbFields[-c(1,2,3)] # Removes UPN and other columns that can't be plotted
```

Comment:
would this be more future-proof if it explicitly listed the fields to exclude (what if column ordering changes?)  Maybe something like:

dbFields <- dbFields[!(dbFields %in% c("UPN","foo","bar"))]

Reply:
Yes, this makes more sense. I made the change.